client/src/App.js

```diff
@@ -20,10 +21,12 @@ export default class App extends Component {
 
   render() {
     return (
-      <div>
-        <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
-      </div>
+        <div>
+          <SavedList list={this.state.savedList} />
+          {/* <Route path='/' exact component={MovieList} /> */}
+          <Route path='/movies/:id' component={MovieCard} />
```

Comment:
You will still want to display the Movie component here - MovieCard will be expecting a prop to render correctly. You should be rendering Movie and passing down the prop to MovieCard through the render method of Movie